app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.players.player import Player
+from app.players.elves.elf import Elf
+from app.players.dwarves.dwarf import Dwarf
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    sum_of_ratings = 0
+    for player in players:
+        sum_of_ratings += player.get_rating()
+    return sum_of_ratings
```

Comment:
Rewrite this method to one line with using function `sum`

Reply:
Done